### DISCUSSION TITLE
about MetPreprocessor input

### DISCUSSION THREAD

**Discussion by zhangzhanggy** [UMEP-dev/UMEP#476] [Q&A]
I'm so sorry about this question, because it seems be apparent.
I just want to confirm that if I can input GHI, DHI and DNI data into MetPreprocessor to get kdown, kdiff and kdir, despite partially satisfying the description and comments.

**Comment by biglimp** [comment]
Yes, it is possible. Please point us to where description is lacking so that we can update.

**Reply by zhangzhanggy** [reply]
thanks, maybe there is nothing lacking, I just was confused about that GHI doesn't seem to be associated with shortwave radiation.
